pkgs/applications/radio/chirp/default.nix

```diff
@@ -1,31 +1,20 @@
-{ stdenv, fetchurl, libxml2Python, libxslt, makeWrapper
-, pyserial, pygtk }:
-
-stdenv.mkDerivation rec {
+{ stdenv
+, fetchurl
+, python2
```

Comment:
Does it not work with Python 3?

Reply:
my best guess is there's a branch with it https://chirp.danplanet.com/projects/chirp/wiki/Linux_Python3.

Reply:
yep https://chirp.danplanet.com/projects/chirp/repository/show?rev=py3

Reply:
The port isn't really complete and has a bunch of issues. Last I checked the first thing I saw was incompatible exception syntax.